sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2214,7 +2221,11 @@ def get_state_port_tbl(self, asic_id):
 
 
 def main():
-    xcvrd = DaemonXcvrd(SYSLOG_IDENTIFIER)
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--skip_cmis_mgr', action='store_true')
```

Comment:
Is this flag supposed to get updated by reading the pmon_daemon_control.json file isn't it?

Reply:
xcvrd gets this arugment from its invocation by supervisord. For eg

python3 xcvrd.py --skip_cmis_mgr